src/views/TenderTable/TenderTable.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div class="tableHeaderContainer"></div>
+  <div class="tenderTable">
+    <div class="tableTitle">
+      <h1> My Tenders</h1>
+    </div>
+    <table style="width: 100%;" v-if="tenders.length !==0">
+      <thead>
+      <tr>
+
+        <th>
+          Field <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Official Name <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Status <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Deadline <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+        <th>
+          Offers <i class="bi bi-sort-alpha-down" aria-label='Sort Icon'></i>
+        </th>
+      </tr>
+      </thead>
+      <tbody>
+
+      <tr  v-for="tender in tenders" :key='tender'>
+        <td style="width: 30%;text-align: center; color: black">{{tender['awardFileKey']}}</td>
```

Comment:
better to create css classes and move styles. by the way I think there is a better way to set column width